src/components/Sing/SequencerPitch.vue

```diff
@@ -466,6 +463,7 @@ onUnmountedOrDeactivated(() => {
   overflow: hidden;
   z-index: 0;
   pointer-events: none;
-  position: relative;
+
+  contain: content;
```

Comment:
ちゃんとわかってないのですが、ここはsizeの封じ込めが大事っぽいので、多分`content+size`なstrictが良いはず･･･！！！
```suggestion
  contain: strict;
```